app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -176,20 +158,33 @@ public int getPlayerId() {
      * The chronometer will countdown from the maximum time of a game to 0
      */
     private void initChronometer() {
+
         chronometer = findViewById(R.id.mapChronometer);
         chronometer.start();
+
         chronometer.setOnChronometerTickListener(chronometer -> {
-                if (chronometerCounter < GAME_TIME) {
-                    chronometerCounter += 1;
-                } else {
-                    Game.endGame(collectedCoins, playerId, MapActivity.this);
-                }
-                chronometer.setFormat("REMAINING TIME "+String.valueOf(GAME_TIME - chronometerCounter));
+            if (chronometerCounter < GAME_TIME) {
+                chronometerCounter += 1;
+            } else {
+                Game.endGame(collectedCoins, playerId, MapActivity.this);
+            }
+
+            chronometer.setFormat("REMAINING TIME " + (GAME_TIME - chronometerCounter));
```

Comment:
It would be a good practice to use a string reference here

Reply:
It's a variable string so I think it might be clearer leaving it like that, I thought about it and that's the conclusion I reached.